pkg/builder/build_client.go

```diff
@@ -13,6 +13,8 @@ import (
 	"github.com/buildbarn/bb-storage/pkg/util"
 	"github.com/golang/protobuf/ptypes"
 	"github.com/golang/protobuf/ptypes/empty"
+	"go.opencensus.io/trace"
```

Comment:
Nit: imports in the Buildbarn tree are grouped by 2nd-level domain name, sorted alphabetically. These two should go below the golang.org ones.

Reply:
That makes more sense, fixed!